user-guide/content-curation/dataset-ingestion/index.qmd

```diff
@@ -12,29 +12,29 @@ from the original datafiles without copies or multiple versions.
 
 For dataset ingestion, generally four steps are required. Depending on the capacity of the dataset provider, some of the steps can be completed by the VEDA team on request.
 
-The data ingestion process requires `Cognito` credentials (username and password). In order to retrieve these credentials, you'll need to contact a member of the VEDA Data Services Team at [veda@uah.edu](mailto:veda@uah.edu) who can set up an account and credentials for you. The first time you log in using the `Cognito Client`, you will be prompted to set a new password.
+The data ingestion process requires `Keycloak` credentials (username and password). In order to set up authentication via Keycloak, see [Setting Up Keycloak](TODO). If you need assistance, you can contact a member of the VEDA Data Services Team at [veda@uah.edu](mailto:veda@uah.edu) who can set up an account and credentials for you. The first time you log in using the `Cognito Client`, you will be prompted to set a new password.
```

Comment:
```suggestion
The data ingestion process requires `Keycloak` credentials (username and password). In order to set up authentication via Keycloak, see [Setting Up Keycloak](CREATE LINK TO KEYCLOAK SETUP INSTRUCTIONS). If you need assistance, you can contact a member of the VEDA Data Services Team at [veda@uah.edu](mailto:veda@uah.edu) who can set up an account and credentials for you. The first time you log in using the `Cognito Client`, you will be prompted to set a new password.
```

This is where I think it would be nice to reference some written instructions for Keycloak @smohiudd